docs/self-managed/tasklist-deployment/tasklist-authentication.md

```diff
@@ -144,6 +149,36 @@ Take the `access_token` value from the response object and store it as your toke
 curl -X POST -H "Content-Type: application/json" -H "Authorization: Bearer <TOKEN>" -d '{"query": "{tasks(query:{}){id name}}"}' http://localhost:8080/graphql
 ```
 
+### User restrictions
+
+When the **User Restrictions** feature is enabled, Tasklist applies additional security measures to filter tasks based on user identity and authorization. The tasks displayed are restricted based on the candidate groups, candidate users, and assignee associated with the logged-in user. The benefits of this resource are:
+
+- Enhanced security: Users only see tasks for which they have the necessary permissions, improving security and preventing unauthorized access.
+- Tasklist customization: The Tasklist interface is tailored to display only relevant tasks for each user, providing a personalized and streamlined experience.
+
+#### Candidate groups
+
+- Tasks will be filtered to include only those associated with candidate groups to which the logged-in user belongs.
+- If a task is configured with candidate groups, only users belonging to those groups will see the task in their task list.
+
+#### Candidate users
+
+- Tasks will be filtered based on candidate users specified for each task.
+- If a task is configured with candidate users, only those users will see the task in their task list.
+
+#### Assignees
+
+- Tasks assigned to a specific user will only be visible to that assigned user and to the users that belong to the candidate groups/users associated with the task.
+- If the logged-in user is the assignee for a task, the task will be displayed in their task list independent of whether the user is on candidate groups/users.
```

Comment:
> If the logged-in user is the assignee for a task, the task will be displayed in their task list independent of whether the user is on candidate groups/users.

Do you mean: if the active user is assigned to a task, the task is displayed in their task list regardless of their candidate group or user status.?